--- ncpfs-2.2.3/lib/Makefile.in.orig	2003-03-13 15:54:46.000000000 +0100
+++ ncpfs-2.2.3/lib/Makefile.in	2004-01-08 14:28:35.000000000 +0100
@@ -54,7 +54,7 @@
 
 BASE_CFLAGS := -DHAVE_CONFIG_H -DMAKE_NCPLIB -D_GNU_SOURCE 
 NDS_CFLAGS := $(BASE_CFLAGS) -DPORTABLE -DUPTON
-CFLAGS_ncplib.o   := -DNWSFIND=\"$(bindir)/nwsfind\"
+CFLAGS_ncplib.o   := -DNWSFIND=\"$(sbindir)/nwsfind\"
 CFLAGS_cfgfile.o  := -DGLOBALCFGFILE=\"$(sysconfdir)/ncpfs.conf\"
 
 STATIC_CFLAGS    := $(CFLAGS) $(CCFLAGS) $(REENTRANT_CFLAGS) @FFUNCTION_SECTIONS@
