--- ncpfs-2.2.6/configure.ac.orig	2007-06-13 02:34:22.000000000 +0200
+++ ncpfs-2.2.6/configure.ac	2007-06-13 02:55:58.000000000 +0200
@@ -909,7 +909,7 @@
 void lsym(void) { };
 void gsym(void) { };
 EOF
-    if ${CC} -shared -Wl,-version-script=conftest.vers -o conftest.${shlibext} conftest.c -lc >/dev/null 2>&1
+    if ${CC} -shared -Wl,-version-script=conftest.vers -o conftest.${shlibext} conftest.c -lc -fPIC >/dev/null 2>&1
     then
       ncp_cv_versioned_link=yes
     fi]
